src/test/resources/unit/multipart_basic

```diff
@@ -0,0 +1,20 @@
+From: Some One <someone@example.com>
+MIME-Version: 1.0
+Content-Type: multipart/mixed;
+        boundary="XXXXboundary text"
+
+This is a multipart message in MIME format.
+
+--XXXXboundary text
+Content-Type: text/plain
+
+this is the body text
+
+--XXXXboundary text
+Content-Type: text/plain;
```

Comment:
shouldn't this appear first in the email in order to test your changes? I think this test passes with the old code, right?

Reply:
Yes, that's right. I added another test -mixed_with_attachments_and_with_body- which test the content-type=text in the second place. 
I made this basic example because the other test cases that do the same build the message with code instead using a mime-type file. 
This test is the basic case.